src/lib/fetch-amis-for-address.ts

```diff
@@ -17,6 +17,25 @@ export default async function fetchAMIsForAddress(
 
   const zip = result.address_components.zip;
 
+  // We pull these from our database (vs geocoder) to ensure a match
+  // when computing whether local incentives are eligible.
+  const supplemental = await db.get<GeoInfo>(
+    `
+    SELECT 
+        city, 
+        county_name as county
+    FROM zips
+    WHERE zip = ?
+  `,
+    zip,
+  );
+  let city: string | undefined;
+  let county: string | undefined;
+  if (supplemental !== undefined) {
```

Comment:
You can just do `if (supplemental)`, or this:

```
const city = supplemental?.city;
const county = supplemental?.county;
```

Reply:
Thanks, using the `?` syntax, I just inlined the city/county assignments when creating the location object below.